### PR TITLE
api call for senior execs

### PR DESCRIPTION
**Pull request by arsidada**

### CODE REVIEW

File: src/Components/Admin/SeniorExecs.js [hunk 1]
```diff
@@ -1,8 +1,144 @@
 import React, {Component} from 'react';
 import {makeStyles} from "@material-ui/core/styles";
+import PropTypes from 'prop-types';
+import { Button } from "@material-ui/core";
+import Box from '@material-ui/core/Box';
+import Collapse from '@material-ui/core/Collapse';
+import IconButton from '@material-ui/core/IconButton';
```
Comment (ahmedhamodi) [inline src/Components/Admin/SeniorExecs.js:7]: Unused imports (reason why Travis CI is failing)

File: src/Components/Admin/SeniorExecs.js [hunk 2]
```diff
@@ -1,8 +1,144 @@
...
+    position,
+    company,
+    approve,
+    deny,
+  };
+}
+
+function Row(props) {
+  const { row } = props;
+  // const [open, setOpen] = React.useState(false);
```
Comment (ahmedhamodi) [inline src/Components/Admin/SeniorExecs.js:37]: Can remove this uncommented line

File: src/Components/Admin/SeniorExecs.js [hunk 3]
```diff
@@ -1,8 +1,144 @@
...
+  const classes = useRowStyles();
+
+  return (
+    <React.Fragment>
+      <TableRow className={classes.root}>
+        <TableCell>
+          {/* <IconButton aria-label="expand row" size="small" onClick={() => setOpen(!open)}>
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </IconButton> */}
```
Comment (ahmedhamodi) [inline src/Components/Admin/SeniorExecs.js:46]: Commented code

File: src/Components/Admin/SeniorExecs.js [hunk 4]
```diff
@@ -1,8 +1,144 @@
...
+                        {Math.round(historyRow.amount * row.price * 100) / 100}
+                      </TableCell>
+                    </TableRow>
+                  ))}
+                </TableBody>
+              </Table>
+            </Box>
+          </Collapse>
+        </TableCell>
+      </TableRow> */}
```
Comment (ahmedhamodi) [inline src/Components/Admin/SeniorExecs.js:90]: I assume this can be removed?

**Review by ahmedhamodi — COMMENTED** [review]
Some initial feedback, not sure if the PR is done/ready for review though.

**Comment by arsidada** [comment]
PR is not ready. I was just using it to look at diffs from master. This is Raho's change that she's using to fetch data about senior executives from Cognito

**Pull request closed without merging**